utils/socket.h

```diff
@@ -0,0 +1,14 @@
+#ifndef UFTRACE_SOCKET_H
+#define UFTRACE_SOCKET_H
+
+#include <sys/socket.h>
+
+#include "uftrace.h"
+
+#define MCOUNT_AGENT_SOCKET_DIR "/tmp/uftrace"
+
+int create_socket(struct sockaddr_un *addr, pid_t pid);
+int listen_socket(int sfd, struct sockaddr_un *addr);
+int send_options(struct uftrace_opts *opts);
```

Comment:
For these external functions, I prefer having a common prefix.  How about `socket_create`, `socket_listen`, `socket_send_options`?  It'd be nice if we could have some comments for them.